cmd/explaintest/r/access_path_selection.result

```diff
@@ -6,39 +6,39 @@ KEY `IDX_b` (`b`),
 KEY `IDX_ab` (`a`, `b`)
 );
 explain select a from access_path_selection where a < 3;
-id	estRows	task	operator info
-IndexReader_6	3323.33	root	index:IndexRangeScan_5
-└─IndexRangeScan_5	3323.33	cop[tikv]	table:access_path_selection, index:a, range:[-inf,3), keep order:false, stats:pseudo
+id	estRows	task	access object	operator info
+IndexReader_6	3323.33	root		index:IndexRangeScan_5
+└─IndexRangeScan_5	3323.33	cop[tikv]	table:access_path_selection, index:IDX_a(a)	range:[-inf,3), keep order:false, stats:pseudo
 explain select a, b from access_path_selection where a < 3;
-id	estRows	task	operator info
-IndexReader_6	3323.33	root	index:IndexRangeScan_5
-└─IndexRangeScan_5	3323.33	cop[tikv]	table:access_path_selection, index:a, b, range:[-inf,3), keep order:false, stats:pseudo
+id	estRows	task	access object	operator info
+IndexReader_6	3323.33	root		index:IndexRangeScan_5
+└─IndexRangeScan_5	3323.33	cop[tikv]	table:access_path_selection, index:IDX_ab(a, b)	range:[-inf,3), keep order:false, stats:pseudo
 explain select a, b from access_path_selection where b < 3;
-id	estRows	task	operator info
-TableReader_7	3323.33	root	data:Selection_6
-└─Selection_6	3323.33	cop[tikv]	lt(test.access_path_selection.b, 3)
-  └─TableFullScan_5	10000.00	cop[tikv]	table:access_path_selection, keep order:false, stats:pseudo
+id	estRows	task	access object	operator info
+TableReader_7	3323.33	root		data:Selection_6
+└─Selection_6	3323.33	cop[tikv]		lt(test.access_path_selection.b, 3)
```

Comment:
why there is a redundant tab`?

Reply:
Because their `access object` is empty. 
I think write nothing instead of "N/A" can make table a liitle cleaner. Because there are so much empty in `AccessObject`.

Reply:
got it.